src/interfacers/EmonHubEmoncmsHTTPInterfacer.py

```diff
@@ -138,5 +151,13 @@ def set(self, **kwargs):
                 self._log.info("Setting " + self.name + " sendstatus: " + setting)
                 self._settings[key] = setting
                 continue
+            elif key == 'compress':
+                self._log.info("Setting " + self.name + " compress: " + setting)
+                if setting == "1":
```

Comment:
`self._cms_settings['compress']` is `1`, an int but this code expects the string `"1"`.